Package.swift

```diff
@@ -3,7 +3,7 @@
 import PackageDescription
 import Foundation
 
-let versionStr = "13.22.0"
```

Comment:
Why 23? There are no breaking changes

Reply:
If there is a breaking change, 13 should become 14
If there are new features: 22 should become 23, 
if there are only fixes: 0 should become 1
Am I wrong?

Reply:
Major version change is only for file format break. Looking at our change log history we only bumped minor part when there is api break, and patch version if there are only fixes and enhancements.

Reply:
from wiki [Releasing-Realm-Core](https://github.com/realm/realm-wiki/wiki/Releasing-Realm-Core):

"Ideally releases from master will contain only bug fixes and as such will bump only the patch part of version number. Releases from develop will bump either the minor part, if adding features, or the major part if breaking anything. That is, if the release requires action on the part of our immediate users (bindings) or their users, the developers (e.g. a required file format upgrade)."

Looks like our documented process is a bit outdated and quite ambiguous. There is new api in current release, but it doesn't break anything for sdks and it doesn't require immediate actions, so it can be used as is. But then again - it's new api. Judging by our changelog.md we were not exactly consistent with minor and patch versioning in last year at least. 

@jedelbo @finnschiermer @ironage how we could clarify our wiki for release process?